src/LINEBot.php

```diff
@@ -95,6 +95,51 @@ public function getProfile($userId)
         return $this->httpClient->get($this->endpointBase . '/v2/bot/profile/' . urlencode($userId));
     }
 
+    /**
+     * Gets the list of User IDs of users who have added your LINE Official Account as a friend.
+     * These users' IDs won't be included in the obtained list of user IDs:
+     * ・ Users who blocked the target LINE Official Account after adding it as a friend.
+     * ・ Users who haven't consented to their profile information being obtained.
+     *
+     * This feature is only available for LINE@ Approved accounts or official accounts.
+     *
+     * @param string $start continuationToken
+     * @return Response
+     */
+    public function getFollowerIds($start = null)
+    {
+        $url = sprintf('%s/v2/bot/followers/ids', $this->endpointBase);
```

Comment:
Could you make url without `sprintf` ?
Like this.
https://github.com/line/line-bot-sdk-php/pull/342/files#diff-22797c186358a12c4ade20a7d7891165463f37cfdf3bb0905ad33893051ac5b5R172

I want to use `sprintf` when is needed to concat 3 or more strings.